### PR TITLE
Updating the Tier preferences with modern format

### PR DESCRIPTION
**Pull request by cartergreen**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/deploy-and-configure-on-macos.md [hunk 1]
```diff
@@ -319,20 +319,30 @@ The example for this setting in the .plist file is:
 
 [More info about configuring the OneDrive sync app for SharePoint Server 2019](/sharepoint/install/new-onedrive-sync-client)
 
+
 ### Tier
+<a name="Tier"> </a>
+
+This setting lets you specify the ring for users in your organization. The OneDrive sync app updates to the public through three rings- first to Insiders, then Production, and finally Deferred.  When you enable this setting and select a ring, users aren't able to change it.
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:326]:
```suggestion
This setting lets you specify the ring for users in your organization. The OneDrive sync app updates to the public through three rings; first to Insiders, then to Production, and finally to Deferred.  When you enable this setting and select a ring, users aren't able to change it.  
```

File: OneDrive/deploy-and-configure-on-macos.md [hunk 2]
```diff
@@ -319,20 +319,30 @@ The example for this setting in the .plist file is:
 
 [More info about configuring the OneDrive sync app for SharePoint Server 2019](/sharepoint/install/new-onedrive-sync-client)
 
+
 ### Tier
+<a name="Tier"> </a>
+
+This setting lets you specify the ring for users in your organization. The OneDrive sync app updates to the public through three rings- first to Insiders, then Production, and finally Deferred.  When you enable this setting and select a ring, users aren't able to change it.  
+
+Insiders - This update ring users receive builds that let them preview new features coming to OneDrive.
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:328]:
```suggestion
**Insiders**: The Insiders ring users receive builds that let them preview new features coming to OneDrive.
```

File: OneDrive/deploy-and-configure-on-macos.md [hunk 3]
```diff
@@ -319,20 +319,30 @@ The example for this setting in the .plist file is:
...
+
+Production - This update ring users get the latest features as they become available. This ring is the default.
+
+Enterprise (now called "Deferred") - This update ring users get new features, bug fixes, and performance improvements last. This ring lets you deploy updates from an internal network location, and control the timing of the deployment (within a 60-day window).
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:332]:
```suggestion
**Enterprise** (now called "Deferred"): The Deferred ring users get new features, bug fixes, and performance improvements last. This ring lets you deploy updates from an internal network location, and control the timing of the deployment (within a 60-day window).
```

File: OneDrive/deploy-and-configure-on-macos.md [hunk 4]
```diff
@@ -319,20 +319,30 @@ The example for this setting in the .plist file is:
...
+
+> [!IMPORTANT]
+> We recommend selecting several people in your IT department as early adopters to join the Insiders ring and receive features early. We recommend leaving everyone else in the organization in the default Production ring to ensure they receive bug fixes and new features in a timely fashion. [See all our recommendations for configuring the sync app](ideal-state-configuration.md)
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:335]:
```suggestion
> We recommend selecting several people in your IT department as early adopters to join the Insiders ring and receive features early. We also recommend leaving everyone else in the organization in the default Production ring to ensure they receive bug fixes and new features in a timely fashion. [See all our recommendations for configuring the sync app](ideal-state-configuration.md).
```

File: OneDrive/deploy-and-configure-on-macos.md [hunk 5]
```diff
@@ -319,20 +319,30 @@ The example for this setting in the .plist file is:
...
+
+For more info on the builds currently available in each ring, see the [release notes](https://support.office.com/article/845dcf18-f921-435e-bf28-4e24b95e5fc0?). For more info about the update rings and how the sync app checks for updates, see the [OneDrive sync app update process](sync-client-update-process.md).
```
Comment (JohanFreelancer9) [inline OneDrive/deploy-and-configure-on-macos.md:337]:
```suggestion
For more information on the builds currently available in each ring, see the [release notes](https://support.office.com/article/845dcf18-f921-435e-bf28-4e24b95e5fc0?). For more information about the update rings and how the sync app checks for updates, see the [OneDrive sync app update process](sync-client-update-process.md).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @cartergreen, please commit my suggested changes if you agree with them. Many thanks.

**Comment by cartergreen** [comment]
I have made all the changes that @JohanFreelancer9 suggested.  Thanks again for the great work.

**Comment by scanum** [comment]
@adeejoseph PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@adeejoseph PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by adeejoseph — APPROVED** [review]

**Pull request merged**